configfiles/Dummy/ToolChainConfig

```diff
@@ -27,6 +27,8 @@ service_kick_sec 60 		      # remove hosts with no heartbeat after given period
 
 ###### Backend Services #####   NEWLY ADDED NEEDS TO HAVE PROPER DESCRIPTIONS AND SOME PRUNING BEFORE RELEASE
 use_backend_services 1		#
+UUID b16a0186-c5df-474c-b880-9fc6c0670d7d  # should this match content of UUID_path
```

Comment:
not sure this is needed for the dummy toolchain thought it just gives you a random UUID if you dont assign one

Reply:
yeah at the time i thought i was having UUID issues, but that turned out to be because of the whole thing of multiple applications using different UUIDs connecting on the same socket.
Still, the question is valid: we have a UUID path, and a UUID variable - how are they connected?